src/data/light.json

```diff
@@ -203,12 +203,12 @@
       },
       "border": {
         "base": {
-          "value": "{colors.grey.400}",
+          "value": "{colors.grey.700}",
           "type": "color",
-          "description": "Used for t4e border colour for primary components such as buttons and Message"
+          "description": "Used for the border colour for primary components such as buttons and Message"
         },
         "hover": {
-          "value": "{colors.grey.500}",
+          "value": "{colors.grey.800}",
           "type": "color",
           "description": "Used for the hovered border colour for primary components such as buttons"
         },
```

Comment:
Maybe I'm using Github wrong but it won't let me comment on code that hasn't been changed.

Let's update the primary.borderActive to be 900 too